src/frontends/tensorflow/tests/convert_tricky_models.cpp

```diff
@@ -280,6 +280,31 @@ TEST_F(TransformationTestsF, ModelWithDilatedGroupConvolution) {
     }
 }
 
+TEST_F(TransformationTestsF, ModelWithSplitConvConcat) {
+    {
+        model = convert_model("split_conv_concat/split_conv_concat.pbtxt");
+        manager.register_pass<pass::MOCTransformations>(false);
+    }
```

Comment:
please put to the compile suite and call `compile_model` for the CPU plugin because we would rather check the whole transformation pipeline.

Reply:
do we test CPU or TF FE here?
we should avoid mixing different components in one test

Reply:
This way we can't detect that someone breaks the whole transformation pipeline. And this is serious thing because it affects performance, if this transformation is not applied properly for this particular model.

Reply:
Why current tests (I mean without compilation) cannot check that transformation pipeline is broken?

Reply:
That is because we need to call `compile_model` in order to have exact transformation pipelin for the plugin.
Otherwise, how can we replicate it? If manually put all transformations executed there, someone can come with an update to common optimizations and CPU plugin pipleline and break it.

Reply:
>  CPU plugin pipleline and break it.

Then the question is: if we test CPU transformation pipeline here, why the test in TF folder?

Reply:
We are testing not only CPU part but also TF FE transformation part. So there are two options at least to put this into CPU tests or TF FE. But FE team is mainly affected by such bugs, so I decided to put this into our suite and maintain such tests.

Reply:
how TF FE can break CPU? 
If after specific commit converted model started to look differently, but it's still valid OpenVINO model => it's CPU issue that it cannot consume this model and has not flexible transformation pipeline.

Reply:
We have multiple transformations inside that can be changed due to optimizations and changing opset.
That is not just a matter of invalid model that the CPU plugin cannot accpet the model. This test is mostly about to preserve true performance optimization when some critical sub-graphs should be transformed.

Reply:
@mateusztabaka, decided with @ilya-lavrenov to have this test in `compilation.cpp` suite to cover the whole pipeline.
Now we have no solution for robust validation of each component (`FE`, `read_model`, `compile_model`). So we have to use joint testing.

Reply:
done